### PR TITLE
Added code to paramertize the libphp prefix

### PR DESCRIPTION
**Pull request by grahamuk2018**
This change allows libphp prefix in the $_lib variable to be changed as required. Currently I need to be able to set the value to libphp- as opposed to libphp.

### CODE REVIEW

File: manifests/mod/php.pp [hunk 1]
```diff
@@ -49,7 +50,7 @@
     $_package_name = undef
   }
 
-  $_lib = "libphp${php_version}.so"
+  $_lib = "$libphp_prefix${php_version}.so"
```
Comment (ekohl) [inline manifests/mod/php.pp]: This should be using braces: `${libphp_prefix}`

**Comment by ekohl** [comment]
Can you also remove the executable bit from the file again?

File: manifests/mod/php.pp (hunk 1 above)
Comment (grahamuk2018) [reply manifests/mod/php.pp]: I have now added the braces.

**Comment by grahamuk2018** [comment]
> Can you also remove the executable bit from the file again?

Sorry but I'm not sure what you mean by this?

**Comment by ekohl** [comment]
> Sorry but I'm not sure what you mean by this?

The file mode changed from 100644 → 100755 which means it's executable. This file shouldn't be.

**Comment by grahamuk2018** [comment]
> > Sorry but I'm not sure what you mean by this?
> 
> The file mode changed from 100644 → 100755 which means it's executable. This file shouldn't be.

Apologies for this, I am using Windows Subsystem for Linux which changes my permissions in certain circumstances. I have now corrected this.

**Comment by grahamuk2018** [comment]
> > > Sorry but I'm not sure what you mean by this?
> > 
> > 
> > The file mode changed from 100644 → 100755 which means it's executable. This file shouldn't be.
> 
> Apologies for this, I am using Windows Subsystem for Linux which changes my permissions in certain circumstances. I have now corrected this.

Thank you :)

**Comment by mmoll** [comment]
this needs a rebase now :)

**Comment by david22swan** [comment]
@grahamuk2018 This change looks good to me. If you could rebase your work and add some documentation and test coverage I would feel happy to merge.

**Comment by lionce** [comment]
Hello @grahamuk2018 ,
It seems like this PR has some conflicts that must be resolved. Look forward to hearing from you.

File: manifests/mod/php.pp [hunk 2]
```diff
@@ -57,7 +58,8 @@
   if $apache::version::scl_httpd_version {
     $_lib = "librh-php${_php_version_no_dot}-php${_php_major}.so"
   } else {
-    $_lib = "libphp${php_version}.so"
+  # Controls php version and libphp prefix
```
Comment (ekohl) [inline manifests/mod/php.pp]: Intending looks off here

**Comment by pmcmaw** [comment]
Merging. All requirements have been met :-) and PR LGTM.
Thanks for the rebase @HelenCampbell 👍

**Pull request merged**